Formula/iam-policy-json-to-terraform.rb

```diff
@@ -0,0 +1,28 @@
+class IamPolicyJsonToTerraform < Formula
+  desc "Convert a JSON IAM Policy into terraform"
+  homepage "https://github.com/flosell/iam-policy-json-to-terraform"
+  url "https://github.com/flosell/iam-policy-json-to-terraform/archive/1.3.0.tar.gz"
+  sha256 "90b5be7caa232186718b5645b2d5ab65a59695854db7598462bdbe059444a051"
+  head "https://github.com/flosell/iam-policy-json-to-terraform.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dir = buildpath/"src/github.com/flosell/iam-policy-json-to-terraform"
+    dir.install buildpath.children
+    cd "src/github.com/flosell/iam-policy-json-to-terraform" do
+      system "make", "vendor", "tools"
+      system "make", "iam-policy-json-to-terraform_darwin"
```

Comment:
What's special about the macOS make?

Reply:
This `make` command runs `GOOS=darwin go build -o $@ *.go` but when I replace this line with

```
system "go", "build", "-o", "iam-policy-json-to-terraform", "*.go"
```

it returns

```
go
build
-o
iam-policy-json-to-terraform
*.go

can't load package: package *.go: cannot find package "*.go" in any of:
	/usr/local/Cellar/go/1.13.7/libexec/src/*.go (from $GOROOT)
	/private/tmp/iam-policy-json-to-terraform-20200209-22723-pqecfk/iam-policy-json-to-terraform-1.3.0/src/*.go (from $GOPATH)

READ THIS: https://docs.brew.sh/Troubleshooting
```

Could you recommend what to do?

Reply:
I dropped the dependency checks since brew takes care of that

```
       system "make", "vendor", "tools" 
```

and kept the original make command that I'm having trouble replicating within homebrew itself.

```
      system "make", "iam-policy-json-to-terraform_darwin"
```

Would this suffice @SMillerDev ?